x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/import_rules/route.test.ts

```diff
@@ -34,6 +34,8 @@ import { importRulesRoute } from './route';
 
 jest.mock('../../../../../machine_learning/authz');
 
+// TODO add tests for connectors
```

Comment:
Are you planning to add them for this PR, or a follow up? 👍 if you'd like to follow-up with an additional test-coverage PR -- would be nice to add a few more FTR tests covering connectors as well.

Reply:
I did already in this file `x-pack/test/detection_engine_api_integration/security_and_spaces/group10/import_rules.ts` which I found exactly the same as this file `x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/import_rules/route.test.ts` and the interesting thing is this file is not referenced at all, that's why I didn't add tests there and I wanted to ask if we need this file actually?

Reply:
So this file contains the jest unit tests against the route, whereas `.../detection_engine_api_integration/security_and_spaces/group10/import_rules.ts` are the FTR (functional test runner) API integration tests that actually stand up a kibana/es instance to test the import e2e.

This file would be good for testing the basic route inputs and such (or any other conditional logic specific to the route itself), but the latter is best for testing the complete flow from request to response, so good to have both. For these unit tests, the only connector specific test you might want to add is around the new `overwrite_action_connectors` query param that was added to the `importQuerySchema`, but as you mention, this could be covered by the FTR tests as well.